File: joao pedro/Problema0405/Problema0405.js
const prompt = require("prompt-sync")({ sigint: true });

var nota=[],tipo=[],resultado=[];

function Media(nota,tipo,resultado){
    
    if(tipo[0].endsWith("P") || tipo[0].endsWith("p")){
    nota[0]=nota[0]*5;
    nota[1]=nota[1]*3;
    nota[2]=nota[2]*2;
    resultado[0]=(nota[0]+nota[1]+nota[2])/10;
}
    else{

        resultado[0]=(nota[0]+nota[1]+nota[2])/3;
    }
}


function Main(){
console.log("Digite 3 numeros: ");
for(i=0;i<3;i++){
nota[i]=Number(prompt(""));}
tipo[0]=prompt("Digite 'A' para media aritmetica ou 'P' para media ponderada: ");



Media(nota,tipo,resultado);
console.log("Resultado: "+resultado[0]);

}

Main();

File: joao pedro/Problema0405/Problema0405.py

notas=[]
tipo=[]
resultado=[]
nota=0
t=""

print("Digite 3 notas: ")
for i in range (3):
        nota=float(input(""))
        notas.insert(i,nota)
print("Digite 'A' para media aritmetica ou 'P' para media ponderada: ")
t=input("")

def Media(notas,t):
    
    if(t.endswith("P") or t.endswith("p")):
        notas[0]=notas[0]*5
        notas[1]=notas[1]*3
        notas[2]=notas[2]*2
        notas[0]=(notas[0]+notas[1]+notas[2])/10

    else:

        notas[0]=(notas[0]+notas[1]+notas[2])/3
    












Media(notas,t)
print("Resultado: ",notas[0])



